src/components/ButtonGroup.scss

```diff
@@ -0,0 +1,106 @@
+// Fonts
+
+@import 'fonts';
+
+.fontButton {
+  font-family: $default-font;
+  font-weight: 300;
+  font-size: 15pt;
+  color: white;
+}
+
+.fontInfo {
+  font-family: $default-font;
+  font-weight: 300;
+  font-size: 12pt;
+  color: black;
+}
+
+// Variables
+
+$button-text: #ffffff;
+$button-student: #4be24f;
```

Comment:
Det kan hende at dette bare er på min skjerm, men er fargene på knappene veldig lyse? Det er bra at de er godt synlige, men jeg synes kanskje at fargene er litt for lyse/kraftige (blir nesten litt vanskelig å se den hvite skriften). Igjen; det kan godt være min skjerm som viser fargene dårlig.